src/api/todos.ts

```diff
@@ -6,3 +6,25 @@ export const getTodos = (userId: number) => {
 };
 
 // Add more methods here
+type Data = {
+  title?: string,
+  userId?: number,
+  completed: boolean,
+};
+
+type ObjData = {
+  id: number,
+  userId: number,
+  completed: boolean,
+  title: string,
+  createdAt: string,
+  updatedAt: string,
+};
```

Comment:
Let's work on naming. Data and ObjData - it's very unclear what these types represent. Too general.